spec/features/unauthenticated_user/unauth_user_admin_views_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature "the unauthenticated user" do
+  scenario "stays on page when attempting to access any /admin route" do
+    visit items_path
+    visit '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.